packages/core/src/components.d.ts

```diff
@@ -8655,6 +8661,10 @@ declare namespace LocalJSX {
         "target"?: BalProps.BalButtonTarget;
     }
     interface BalNavbarMenu {
+        /**
+          * @default false
+         */
+        "active"?: boolean;
```

Comment:
Do we need this? Looks like as we do not use it.

Reply:
generated by stencil